analytics/terraform/spark-eks-ipv6/vpc.tf

```diff
@@ -0,0 +1,33 @@
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 5.0"
+
+  name = local.name
+  cidr = local.vpc_cidr
+
+  azs             = local.azs
+  private_subnets = [for k, v in local.azs : cidrsubnet(local.vpc_cidr, 4, k)]
+  public_subnets  = [for k, v in local.azs : cidrsubnet(local.vpc_cidr, 8, k + 48)]
+
+  enable_nat_gateway     = true
+  single_nat_gateway     = true
+  enable_ipv6            = true
+  create_egress_only_igw = true
+
+  public_subnet_ipv6_prefixes                    = [0, 1, 2]
```

Comment:
Since its an iPV6 Spark blueprint,. It would be to highlight the code Terraform and Spark config as well.
e.g., iPV6 config for Subnets or ipV6 config for Spark etc as a comment to just above the code block